src/sentry/utils/http.py

```diff
@@ -214,15 +216,14 @@ def is_valid_ip(ip_address, project):
     Verify that an IP address is not being blacklisted
     for the given project.
     """
-    blacklist = project.get_option('sentry:blacklisted_ips')
+    blacklist = project.get_option('sentry:{}'.format(FilterTypes.BLACKLISTED_IPS))
     if not blacklist:
         return True
 
     for addr in blacklist:
-        # We want to error fast if it's an exact match
+        # we want to error fast if it's an exact match
```

Comment:
😬  it's a result of me refactoring stuff unnecessarily and then attempting to put this function back into its original form